### PR TITLE
make template

### PR DESCRIPTION
**Pull request by SOTAkkkk**
#やったこと

- READMEの更新
- 雛形を作成

### CODE REVIEW

**Review by a163236 — CHANGES_REQUESTED** [review]
Cloud you remove .DS_Store and .gitignore ?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.gitignore
```
Comment (a163236) [inline .gitignore]: it must be not .gitignore but .DS_Store, isn't it ?

Comment (SOTAkkkk) [reply .gitignore]: tesi

**Comment by SOTAkkkk** [comment]
please check my commit

**Review by a163236 — APPROVED** [review]
popo

**Pull request merged**